src/hooks/useWalletSectionsData.ts

```diff
@@ -27,6 +29,13 @@ export default function useWalletSectionsData({
   } = useLegacyNFTs({
     address: accountAddress,
   });
+
+  const walletsWithBalancesAndNames = useWalletsWithBalancesAndNames();
+
+  const accountWithBalance = walletsWithBalancesAndNames[selectedWallet.id]?.addresses.find(
+    address => address.address.toLowerCase() === accountAddress.toLowerCase()
+  );
+
```

Comment:
In order to keep the data displayed on the wallet screen consistent with the rest of the app, we need to consume this hook here.